app/assets/stylesheets/layout.scss

```diff
@@ -62,6 +62,14 @@ body{
     background: $main-panel-background;
     margin: 1em;
 
+    a:hover{
+      background-color: inherit;
+    }
+
+    .field_with_errors{
+      background: #ff9999;
```

Comment:
Rails-native, it's just `red;` e.g. `#f00`, which isn't a good contrast ratio with black text (5.3:1, so passes AA but fails AAA). This is 10.3:1, which passes WCAG 2.0 AAA as well. 

Do we think I should write this as `#f99`, btw?